scripts/utils/import_languages.py

```diff
@@ -165,7 +165,10 @@ def qtquery(qmake, propname):
 
 # Step 4
 title("Step 4", "Generate the Js/C++ string definitions...")
-generate_strings.generateStrings()
+stringdict = GS.parseTranslationStrings('translations/strings.yaml')
```

Comment:
Why these changes? In this way, we are propagating the string complexity to other files. Is it not better to keep it all in the `GS` file?

Reply:
The CMake build process isn't using `import_languages.py` at all, I'm just trying to keep it up to date with the refactoring in `generate_strings` so that it doesn't break. I plan on removing the script entirely at some point, so I'm just not being very tidy about it. I suppose it might be a little cleaner to run `generate_scripts.py` as a subprocess and pass it arguments, like are doing in `translations/CMakeLists.txt`

The changes to `generate_strings.py` are necessary because CMake kind of expects you to support out-of-tree builds, so it becomes necessary to ensure we can be a bit more flexible with the input and output file locations.